### PR TITLE
Add responses to JSON study

### PR DESCRIPTION
**Pull request by lisamsmith100**
by Lisa Smith

### CODE REVIEW

**Comment by lisamsmith100** [comment]
I see now that I should have written out the syntax although with modeling, I was just 'modeling' it.  In future, will use the curly braces, brackets, colons, quotes, commas.  I ran out of time on this but wanted to demonstrated that I understand the hierarchy.

File: study.md [hunk 1]
```diff
@@ -50,7 +50,25 @@ A kitchen has many items.
...
+          extra large
+      rack
+        herbs
+          chives
+          lemongrass
+          mint
+        spices
+          cinnamon
+          nutmeg
+          vanilla-->
```
Comment (jordanallain) [inline study.md:71]: Hey Lisa we wanted you to structure it like JSON which is basically just like a JavaScript object. Something more like this:

```
{ kitchen: {
    cupboard: [
      'bowls',
      'plates'
    ]
  }
}
```

**Comment by lisamsmith100** [comment]
Thanks!  I understand it now.  Thanks for the note ☺

**Comment by jordanallain** [comment]
Hey saw your note about not realizing you were supposed to write out the correct syntax. Otherwise thanks for doing the study.

👍 🇯🇴

**Pull request closed without merging**